compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Utils/AMDAIEDmaUtils.cpp

```diff
@@ -292,6 +304,60 @@ LogicalResult combineAccessPatterns(
   return success();
 }
 
+LogicalResult expandLargeDimIntoLinearDims(
```

Comment:
It's better to add some documents for this function.

Reply:
Done